docs/content/users/quickstart.md

```diff
@@ -149,6 +149,40 @@ While the generic `php` project type is [ready to go](./project.md) with any CMS
         ddev launch
         ```
 
+=== "ExpressionEngine"
+
+    ## ExpressionEngine
+
+    === "ExpressionEngine Zipfile Download"
```

Comment:
Every dictionary I found agrees it’s two words, and it wouldn’t hurt to match ExpressionEngine’s own docs here:

```suggestion
    === "ExpressionEngine ZIP File Download"
```

From [this page](https://docs.expressionengine.com/latest/installation/installation.html):

![Screen Shot 2023-03-21 at 06 04 36 PM@2x](https://user-images.githubusercontent.com/2488775/226775600-7e5d0bae-1edf-491d-a3ab-6a31c435e1b0.png)